File: Flaadestation.API/Flaadestation.Repository/Database/Entities/License.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flaadestation.Repository.Database.Entities
{
    public class License
    {
        public Guid LicenseId { get; set; }
        public Guid LicenseKey { get; set; }
        public DateTime ValidFrom { get; set; }
        public DateTime ValidTo { get; set; }
        public Guid CompanyId { get; set; }
        public string? UserId { get; set; }

        public Company? Company { get; set; }
        public ApplicationUser? User { get; set; }
    }
}

File: Flaadestation.Dashboard/src/Models/License.ts
import type { Company } from "./Company";
import type { User } from "./user";

export interface License {
    licenseId: string;
    licenseKey: string;
    validFrom: string;
    validTo: string;
    user?: User;
    company: Company
}

export class LicenseModel {
    license: License;

    constructor(license: License) {
        this.license = license;
    }

    get data(): License {
        return this.license;
    }
}